internal/store/driver.go

```diff
@@ -568,7 +701,74 @@ func (d *Driver) calculateHTTPSEdges() map[string]ingressv1alpha1.HTTPSEdge {
 		edge.Spec.Metadata = d.customMetadata
 		edgeMap[domain.Spec.Domain] = edge
 	}
+	d.calculateHTTPSEdgesFromIngress(edgeMap)
+
+	if d.gatewayEnabled {
+
+		httproutes := d.store.ListHTTPRoutes()
+		gateways := d.store.ListGateways()
+		for _, gtw := range gateways {
+			var gatewayDomains []string
+			for _, listener := range gtw.Spec.Listeners {
+				if listener.Hostname == nil {
+					continue
+				}
+				gatewayDomains = append(gatewayDomains, string(*listener.Hostname))
+			}
+			if len(gatewayDomains) <= 0 {
```

Comment:
just `== 0`?

Reply:
late night coding lets in the silliest mistakes 😆

Reply:
fixed here: https://github.com/ngrok/kubernetes-ingress-controller/pull/343/commits/e8a6d917d85cbfe3bdd5d45362836bebb99288ba